zerver/tests/test_channel_folders.py

```diff
@@ -10,6 +11,29 @@
 from zerver.models.streams import get_stream
 
 
+class ChannelFoldersTestCase(ZulipTestCase):
+    @override
+    def setUp(self) -> None:
+        super().setUp()
+        zulip_realm = get_realm("zulip")
+        iago = self.example_user("iago")
+        desdemona = self.example_user("desdemona")
+        lear_user = self.lear_user("cordelia")
+
+        check_add_channel_folder(
+            zulip_realm, "Frontend", "Channels for frontend discussions", acting_user=iago
+        )
+        check_add_channel_folder(
+            zulip_realm, "Backend", "Channels for **backend** discussions", acting_user=iago
+        )
+        check_add_channel_folder(
+            zulip_realm, "Marketing", "Channels for marketing discussions", acting_user=desdemona
+        )
+        check_add_channel_folder(
+            lear_user.realm, "Devops", "Channels for devops discussions", acting_user=lear_user
+        )
```

Comment:
Generally this sort of test setup method isn't a favored pattern -- having setup in `SetUp` methods like this is a lot less readable than just having a function that you call, since the body of the test itself won't have any hint as to how these got created.

(And further, this work will be done for every test in the class, even if the test grows and doesn't use these)